web-ui/src/main/resources/catalog/components/edit/onlinesrc/partials/onlinesrcList.html

```diff
@@ -127,30 +127,48 @@
           <span data-translate="">add</span>
         </a>
       </h2>
+
+      <div class="row">
+        <span data-translate="">sortLinksBy</span>
+        <a class="btn btn-link"
+           data-ng-repeat="s in sortLinksOptions"
+           data-ng-click="sortLinksBy(s)">
+          <span>{{('link-' + s) | translate}}</span>
+
+          <i class="fa"
+             data-ng-show="sortLinksProperty === s"
+             data-ng-class="{'fa-caret-down': !sortLinksReverse, 'fa-caret-up': sortLinksReverse}"></i>
+        </a>
+
+        <input type="text" class="form-control input-sm"
+               placeholder="{{'filter' | translate}}"
+               data-ng-show="relations.onlines.length > 9"
+               data-ng-model="sortLinksFilter">
+      </div>
       <ul class="list-group"
           data-ng-show="::isCategoryEnable('onlinesrc') && relations.onlines && relations.onlines.length > 0">
         <li class="list-group-item" href=""
-            data-ng-repeat="resource in relations.onlines | orderBy:sortLinks">
-          <div class="row">
-            <!--Specific geocat: use lUrl instead of url-->
+            data-ng-repeat="resource in relations.onlines | filter:sortLinksFilter | orderBy:sortLinks:sortLinksReverse">
+          <div class="row"
+	       data-ng-show="relations.onlines.length > 1">
```

Comment:
Not sure if the issue described in https://github.com/geonetwork/core-geonetwork/pull/3804#issuecomment-501301126, can be related to this check?

Reply:
Fixed.